exchanges/order/orders.go

```diff
@@ -676,3 +676,30 @@ func (o *ClassificationError) Error() string {
 		o.Exchange,
 		o.Err)
 }
+
+// Validate checks internal struct requirements
+func (o *Cancel) Validate() error {
```

Comment:
Please add a check here to ensure that o.ID is set

Reply:
🦅 👍

Reply:
`CancelAllOrders` wrapper func shares the same `Cancel` struct, which this change may impact. So you may need to make a new struct for cancelAllOrders, or something along those lines

Reply:
![image](https://user-images.githubusercontent.com/9391715/94087364-e3beea00-fe50-11ea-8db5-a4fc00ac683d.png)